Cargo.toml

```diff
@@ -9,7 +9,8 @@ authors = [
 license = "MIT"
 
 [features]
-default = ["protobuf", "tower-h2"]
+default = ["protobuf", "tower-h2", "hyper-body"]
+hyper-body = ["hyper"]
```

Comment:
I'd prob just call this `hyper` that aligns better with the other trait names.

Reply:
Can't because hyper is a dependency name

Reply:
All optional dependencies are automatically features, so you don't need to explicitly list out the feature.

Reply:
Ahh okay so I can just take it out entirely

Reply:
Yes, but see the comment below first.